pom.xml

```diff
@@ -136,6 +136,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
if we add the spotless plugin, all Maven plugins will be sorted alphabetically here too. So you can already put this maven-release-plugin at the (approximately) correct alphabetic position in the plugin list to make that diff smaller then. ;-)

Reply:
The rest of the plugins are so badly ordered in terms of alphabetical order, so I don't think that this particular positioning will single-handedly ruin the diff. I added it to the end, because I didn't know where else to add it.